File: Kotlin/src/main/kotlin/d_while_loops/exercise_40.kt
package d_while_loops

// 40) Create an application that displays the following count on the screen: 0 3 6 9 12 15 18 Finished!

fun main() {
    var counter = 0

    while (counter <= 18) {
        print("$counter.. ")
        counter+=3
    }
    print("Done!")
}

File: Dart/d_while_loops/exercise_40.dart
// 40) Create an application that displays the following count on the screen:
// 0 3 6 9 12 15 18 Finished!

import 'dart:io';

void main() {
  List<int> vector = [];
  int counter = 0;
  int maxSize = 18;
  int change = 3;

  while (counter <= maxSize) {
    vector.add(counter);
    counter += change;
  }

  for (int element in vector) {
    if (element < maxSize) {
      stdout.write('$element, ');
    } else {
      stdout.write('$element. ');
    }
  }
  print('finished!');
}
